sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -202,6 +202,28 @@ Azure::Security::KeyVault::Keys::DeleteKeyOperation KeyClient::StartDeleteKey(
           {_detail::KeysPath, name}));
 }
 
+Azure::Security::KeyVault::Keys::DeleteKeyOperation KeyClient::ResumeDeleteKey(
+    std::string const& resumeToken,
+    Azure::Core::Context const& context) const
+{
+  Azure::Security::KeyVault::Keys::DeleteKeyOperation operation(m_pipeline, resumeToken);
+  // Need to fix this to use context directly. See:
+  // https://github.com/Azure/azure-sdk-for-cpp/issues/2091
+  operation.Poll(context.GetApplicationContext());
```

Comment:
We are assuming here that `Poll` will set the status and raw response correctly, right?